### ISSUE TITLE
Long request times

### ISSUE THREAD

**Issue by Aimtracker** [TheCookieLab/poloniex-api-java#7]
Hey, I am currently writing a bot and it automatically requests the chart Data every 5 seconds.
Usually it works but after the 5th request (sometimes more, sometimes less) it takes up to 30 seconds to get a response.
Another thing I noticed was, that if I try to monitor 2 different currency pairs, that I only get information about the second currency pair.
Do you know a solution for this, or am I missing something?

**Comment by TheCookieLab** [comment]
I'm not in front of my computer at the moment so can't answer definitively but the sporadic slowness might just be due to load at Poloniex. Regarding the multiple calls, it could be a concurrency issue. Try instantiating two PoloniexExchangeService objects and see if it still occurs. Taking a step back though, this type of use case is probably best handled with the websocket interface (just merged with master a couple days ago)

**Comment by thiko** [comment]
Hi, I'm currently not using that project to poll data from poloniex but I've observed the same behavior by using my own python implementation. In my case, I sent requests for 5 different currency pairs (in a row, not concurrent). Some of that requests fails sometimes... when I tried the same request again, it was mostly working.
Sounds stupid but the only "solution" without (web-)sockets was a retry... 

Cheers!
